styles.css

```diff
@@ -48,11 +95,96 @@ body {
   display: none;
   position: absolute;
   background-color: white;
-  min-width: 160px;
+  width: 375px;
   box-shadow: 0 8px 16px 0 rgba(0, 0, 0, 0.2);
   z-index: 1;
 }
 
+/* Menu */
+.menu-container {
+  position: fixed;
+  top: 0;
+  left: 0;
+  width: 0%;
+  height: 100%;
+  z-index: 1;
+  mix-blend-mode: multiply;
+  background: #6070ff;
+}
+
+.menu-close-container {
+  display: flex;
+  justify-content: flex-end;
+  padding: 16px 38px 32px 0;
+}
+
+.close-icon {
+  display: flex;
+  justify-content: flex-end;
+  padding-right: 100px;
+  padding-top: 24px;
+}
```

Comment:
- [ Optional ] Kindly I recommend you stick to the design as much as possible ( margins) using [the templates in Figma](https://www.figma.com/file/l7SqJ3ZfkAKih9sFxvWSR4/Microverse-Student-Project-1?node-id=0%3A1). Please adjust the right margin of the close menu and bottom margin of overlay just like the figma design.

Your Mobile Menu Look likes this : 
![Capture](https://user-images.githubusercontent.com/93598189/210354670-ddc19f3a-32f6-45fb-ab44-009d583e1ae2.PNG)

The Figma Mobile Menu Look likes this : 

![Capture](https://user-images.githubusercontent.com/93598189/210354851-ac1ee12d-95d3-4859-81f5-789fdb89be58.PNG)